### DISCUSSION TITLE
Move a Placement by Hotkey/Modifier?

### DISCUSSION THREAD

**Discussion by badmin-c** [maruohon/litematica#791] [Q&A] [answered]
Hi,
do we have a way to move a placement block by block by using a Modifier or Hotkey, without having to enter the menu and clicking +/- X/Y/Z?

**Comment by maruohon** [comment, marked as answer]
Yes, you can hold the `selectionNudgeModifier` hotkey (default is `Left Alt`) and scroll with the mouse, while looking in the direction you want to move the selected element to. "Selected element" here refers to any movable and selectable element within the mod, which can be an entire placement, a placement sub-region, an area selection sub-region box, corner, the manual origin, or the entire multi-region selection.

The quickest way to select elements usually is to use the `toolSelectElements` hotkey (which is by default mouse middle click) while looking at the element while holding the tool item. The maximum range for selecting stuff is something around 200 blocks away I think, so you don't need to be right next to the thing to select it, usually you can select it from as far away as you can reasonable see and target the desired element.

To select a sub-region instead of the entire placement you need to also hold the `selectionGrabModifier` key. (This is kind of a relic and this key and some others will be changed/clarified during the mod rewrites and clean-up.)

You can also select things via the Schematic Placements menu (click on a placement so it gets the white outline), the placement configuration menu (to select sub-regions), the Area Editor menu (to select sub-regions or corners).

**Reply by badmin-c** [reply]
That's exactly what I was looking for. Thank you for the quick reply!

**Reply by OhjayMaybe** [reply]
hey, i know this is very old but when i do left alt + scroll wheel it resizes the box instead of moving it. i'm on option 7/9 (move), and i dont know what else i can do, can someone help?

**Reply by maruohon** [reply]
By "resize" do you mean it moves one of the corners, or does it actually grow/shrink the entire box from all sides? In the latter case you would probably have rebound the `selectionGrowModifier` hotkey to just Left Alt.

If it just moved the selected corner, then that's intended if you have selected one of the corners. If you instead select the box but not a corner (by middle clicking with the tool while looking at the box elsewhere than on the two colored corners), then it should nudge the entire box.

And lastly, I do not recommend using the creative mode Move tool mode (7/9) basically ever, as it's so finicky to use. If you want to move stuff in the world, then I'd recommend just cloning the build and then deleting the old build instead. This can be done using the `cloneSelection` hotkey as a shortcut, and then using the Delete tool mode.

And be very careful with all of these creative mode features, as there is no undo, and some of the modes like Delete have a sub-mode, which determines whether you will delete the current area selection contents, or the current schematic placement contents. Always look at the tool HUD before executing any creative mode operations, to confirm that you are in the correct mode and sub-mode, and have the correct area or placement selected etc. You toggle between the sub-modes by default with `Ctrl + M`.
